EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,107 @@
+import datetime
+from typing import Callable, Optional
+from time import time
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
```

Comment:
Surely for this you would want .timestamp()?

Reply:
let it be